trade-finance-manager-ui/server/nunjucks-configuration/index.js

```diff
@@ -24,6 +29,13 @@ const configureNunjucks = (opts) => {
 
   const nunjucksEnvironment = nunjucks.configure(appViews, opts);
 
+  // TODO: DTFS2-7138 remove contact us env vars
+  nunjucksEnvironment.addGlobal('CONTACT_US_SELF_SERVICE_PORTAL_URL', CONTACT_US_SELF_SERVICE_PORTAL_URL);
+  nunjucksEnvironment.addGlobal(
+    'CONTACT_US_EMAIL_ADDRESS',
+    CONTACT_US_EMAIL_ADDRESS || 'DigitalService.TradeFinance@ukexportfinance.gov.uk',
```

Comment:
Can we remove `'DigitalService.TradeFinance@ukexportfinance.gov.uk'` please?

Reply:
sorry other comment got lost -- we should keep this tohave a fall back in case env vars are missing. we have a ticket to remove this once we've migrated over